core/src/main/java/com/netflix/conductor/auth/AuthManager.java

```diff
@@ -70,6 +72,8 @@
 	private String authEndpoint;
 	private boolean traceIdEnabled;
 
+	private final AtomicReference<Map<String, Object>> cachedAuthDataRef = new AtomicReference<>();
```

Comment:
Do we need map ? Can't we use tuple.Pair<> from google common lib. I think it is already part of libraries

Reply:
I decided to make it flexible in order to have a possibility to add another information if needed